src/js/actions/tools.js

```diff
@@ -200,6 +205,89 @@ define(function (require, exports) {
     };
 
     /**
+     * Resets the pointer policies around the selection border so we can pass
+     * pointer events to Photoshop for transforming, but still be able to
+     * click through to other layers
+     *
+     * @return {Promise}
+     */
+    var resetBorderPoliciesCommand = function () {
+        var toolStore = this.flux.store("tool"),
+            appStore = this.flux.store("application"),
+            uiStore = this.flux.store("ui"),
+            currentDocument = appStore.getCurrentDocument(),
+            currentTool = toolStore.getCurrentTool();
+
+        // We only want to reset superselect tool
+        if (!currentDocument || !currentTool || currentTool.id !== "newSelect") {
+            return Promise.resolve();
+        }
+
+        var targetLayers = currentDocument.layers.selected,
+            artboards = targetLayers.some(function (layer) {
+                return layer.isArtboard;
+            }),
+            selection = currentDocument.layers.selectedAreaBounds;
+
+        if (!selection || selection.empty) {
+            return Promise.resolve();
+        }
+
+        var psSelectionTL = uiStore.transformCanvasToWindow(
+                selection.left, selection.top
+            ),
+            psSelectionBR = uiStore.transformCanvasToWindow(
+                selection.right, selection.bottom
+            ),
+            psSelectionWidth = psSelectionBR.x - psSelectionTL.x,
+            psSelectionHeight = psSelectionBR.y - psSelectionTL.y,
+            // The resize rectangles are roughly 10 points radius
+            inset = 10,
+            // In case of artboards, we have no rotate, so we can stay within the border
+            outset = artboards ? inset : 25;
+
+        var insidePolicy = new PointerEventPolicy(adapterUI.policyAction.NEVER_PROPAGATE,
```

Comment:
A little comment about what you're doing here geometrically might be useful to the next schmuck to come across this.